railties/lib/rails/generators/rails/app/templates/docker-entrypoint.tt

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
Does this pass shellcheck? I thought they prefer using `set` in separate statements, but I could be wrong. Will investigate. 👀